src/declarations/stencil-private.ts

```diff
@@ -196,13 +197,24 @@ export interface RollupResults {
   modules: RollupResultModule[];
 }
 
+export interface UpdatedBuildCtx {
```

Comment:
Can we rename this please? I don't love the idea of having an interface named `UpdatedBuildCtx` who isn't used across all the different build types that we have today (e.g. `custom-dist`)

Reply:
Could you recommend a better name?